### DISCUSSION TITLE
compatible esp-01 ?

### DISCUSSION THREAD

**Discussion by mars-eyes** [softwarecrash/Victron2MQTT#23] [Q&A]
fonctionne sur wemos d1 est il possible de le compiler sur un esp01 qui n'utilise que du 3.3v avec 1mo de stockage ?
associé à un 100/20- 12/48v

**Comment by all-solutions** [comment]
Please in English in the future ;-)

In principle, that would be possible. However, we currently have no free capacity for this. And this will continue for the next weeks.

If you have time, we can put this on the to-do list for future releases. But when and in which release this will come... is open.

**Comment by mars-eyes** [comment]
Thanks for your feedback
I have several esp-01 and wemos d1 lying around so I'm in no rush!
But I think the esp-01 would be able to work at a lower cost
